File: check_palindrome.py
def checkPalindrome(inputString):
    # iterate from left to right, store items in list 
    # iterate from right to left, store items in list
    # compare lists 
    left_to_right = list(inputString)
    right_to_left = list(reversed(inputString))
    print(left_to_right)
    print(right_to_left)
    
    if left_to_right == right_to_left:
        return True
    else:
        return False 

def main():
    i_string = input()
    print(checkPalindrome(i_string))

if __name__ == "__main__":
    main()

File: CheckPalindrome.java
import java.util.Scanner; 
import java.util.Arrays; 

public class CheckPalindrome {

    public static void main(String[] args) {
        Scanner n = new Scanner(System.in); 
        String palindrome_candidate = n.nextLine(); 
        System.out.println(checkPalindrome(palindrome_candidate)); 
    }

    static boolean checkPalindrome(String inputString) {
        char[] left_to_right = inputString.toCharArray();
        char[] right_to_left = new char[inputString.length()];

        int j = 0;
        for (int i = inputString.length() - 1; i >= 0; i--) {
            right_to_left[j] = left_to_right[i];
            j++; 
        }

        System.out.println(right_to_left);
        if (Arrays.equals(left_to_right, right_to_left)) {
            return true; 
        }
        else {
            return false; 
        }
    }
}